CHANGELOG.md

```diff
@@ -90,8 +108,10 @@ All notable changes to the Zowe API Mediation Layer package will be documented i
 * Bugfix:  snakeyml update, scheme validation fix (#2577) ([ae48669](https://github.com/zowe/api-layer/commit/ae48669)), closes [#2577](https://github.com/zowe/api-layer/issues/2577)
 * Bugfix:  Add build info to the manifest.yaml (#2573) ([93298dd](https://github.com/zowe/api-layer/commit/93298dd)), closes [#2573](https://github.com/zowe/api-layer/issues/2573)
 * Bugfix:  Fix bug in the swagger (#2571) ([36997c6](https://github.com/zowe/api-layer/commit/36997c6)), closes [#2571](https://github.com/zowe/api-layer/issues/2571)
+* Bugfix:  Handle exceptions in extensions config reader (#2609) ([336d3b4](https://github.com/zowe/api-layer/commit/336d3b4)), closes [#2609](https://github.com/zowe/api-layer/issues/2609)
```

Comment:
This isn't clear as to what the actual fix was. It'd be easier to understand if bug fixes were written in past tense.

Reply:
The reason is to clean up the log. It was full of non-helpful messages. So, something like `...to avoid fulfilling log with unexpected messages.`? Anyway, it is not the current changelog. Does it make sense to update it now?